kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
The more I think about it, the more I think we should take it out now. I think way to include this would be:

1. Write a 'LabelSelector' trait type, that can be used to express either matchLabels or matchExpressions
2. Use that for this and for nodeSelectors

I think that's the cleanest and most future compatible way of doing this. what do you think?